examples/resources/aquasec_role_mapping_saas/resource.tf

```diff
@@ -0,0 +1,8 @@
+resource "aquasec_role_mapping_saas" "aquasec_roles_mapping_saas" {
+  saml_groups = ["group1", "group2"]
+  csp_role = "Administrator"
+}
+
+output "ssss" {
```

Comment:
`roles_mapping_saas` or `aquasec_roles_mapping_saas` would be better than `ssss`

Reply:
Yes, it by mistake, I will fix the pr